### ISSUE TITLE
zip I/O Error

### ISSUE THREAD

**Issue by katmoody** [humanmade/backupwordpress#986]
This has been coming up frequently, so listing as a bug but please correct if it means something else? 

```
Zip: zip I/O error: No such file or directory, zip error: Could not create output file (was replacing the original zip file)
```

**Comment by pdewouters** [comment]
just got it on my local install, I'll try and debug, although it's probably related to https://github.com/humanmade/backupwordpress/issues/970

**Comment by pdewouters** [comment]
hmm, weird. I can't reproduce it anymore

**Comment by katmoody** [comment]
In SOME cases, this indicates a larger inability to actually back up their sites; in everyone else it's a nuisance error. 

Clear as mud, right? :D

**Comment by willmot** [comment]
@kat in situations where this error occurs the backup should still work, `zip` would fail and the plugin should then just fallback to `zipArchive`. If that's not happening then if we could get access to someones site where this is reproducible that would help a lot

**Comment by kat** [comment]
I think you have meant @katmoody : in situations where this error occurs the backup should still work, zip would fail and the plugin should then just fallback to zipArchive. If that's not happening then if we could get access to someones site where this is reproducible that would help a lot

**Comment by willmot** [comment]
Sorry @kat! 😊

**Comment by willmot** [comment]
Do we have a way to reproduce this error reliably? There's not enough to go on at the moment.

There's two parts to this that I'm interested in.

1. What is causing that `zip` error? Is it something we have control over?
2. When that error happens why does the backup just fallback to `ZipArchive` and complete successfully? If it does then we probably just shouldn't be surfacing that error. If it doesn't then why not?

**Comment by willmot** [comment]
Dug into this and into point 2. above, found the issue. We weren't correctly checking errors when verifying that the backup had completed successfully. Since 9dbcac6 I'm now almost certain this is going to be fixed in #1001
